modules/nf-core/openms/filefilter/main.nf

```diff
@@ -0,0 +1,51 @@
+process OPENMS_FILEFILTER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/openms:3.2.0--haddbca4_4' :
+        'biocontainers/openms:3.2.0--haddbca4_4' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), path("*.{mzML,featureXML,consensusXML}"), emit: filtered
```

Comment:
Can you separate these files into different output channels?

Reply:
I could, but don't you think that suggests the tool outputs multiple files?

Reply:
No the [guidelines](https://nf-co.re/docs/guidelines/components/modules#one-output-channel-per-output-file-type) state that each output file should have it's own output channel